### PR TITLE
Add a CODEOWNERS file

### PR DESCRIPTION
**Pull request by theospears**
## Summary
It's becoming a little hard for me to keep track of PR states. To help with this, I'm going to experiment with configuring github to request a review from me on all PRs. Then I can filter to PRs waiting on review from me to know which need my attention.

CODEOWNERS appears the simplest way to do this.

## Validation
As this is a git change it's hard to verify pre-merge. Will merge, and then check PRs start listing me as a reviewer.

### CODE REVIEW

**Comment by theospears** [comment]
* **#570** <a href="https://app.graphite.dev/github/pr/beeminder/BeeSwift/570?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/beeminder/BeeSwift/570?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `master`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Request reviews from the iOS app maintainer by default
+* @beeminder/ios-maintainer
```
Comment (krugerk) [inline .github/CODEOWNERS:2]: this team has to be created elsewhere, which I am guessing you did


Is it named "a team"? Maybe Beeminder Maintainer or BeeSwift Maintainers could be the name?

Screenshot 2024-12-29 at 17 50 19

Comment (theospears) [reply .github/CODEOWNERS:2]: It's called `ios-maintainer`. Unfortunately github doesn't support making it public: https://github.com/orgs/community/discussions/20808

Comment (theospears) [reply .github/CODEOWNERS:2]: This is how those lines show up for me.

image

**Pull request merged**